src/styles/styles.js

```diff
@@ -1239,6 +1239,14 @@ const styles = {
         fontSize: variables.iouAmountTextSize,
     }, 0),
 
+    autoGrowTextInputHiddenText: {
+        position: 'absolute',
+        opacity: 0,
+
+        // Using a very large value here makes sure it's out of screen
+        left: 100000,
```

Comment:
I'm not really getting the hidden text thing sorry. IMO we should avoid using values like this. If we absolutely must push something off screen we can at least base this value on the screen width + current position of element.